galaxy_ng/app/api/ui_v2/views.py

```diff
@@ -0,0 +1,41 @@
+from rest_framework import viewsets
+from rest_framework.response import Response
+from rest_framework.decorators import action
+from django_filters.rest_framework import DjangoFilterBackend
+
+from ansible_base.resource_registry.models import Resource
+from ansible_base.rest_pagination.default_paginator import DefaultPaginator
+
+from .filters import UserResourcesViewFilter
+from .filters import OrganizationResourcesViewFilter
+from .filters import TeamResourcesViewFilter
+from .models import UserResourcesView
+from .models import OrganizationResourcesView
+from .models import TeamResourcesView
+from .serializers import UserSerializer
+from .serializers import OrganizationSerializer
+from .serializers import TeamSerializer
+
+
+class UserViewSet(viewsets.ModelViewSet):
```

Comment:
All of this stuff here is unnecessary (except for the filter). We can add the resource obj on the serializer like so:

```diff
diff --git a/galaxy_ng/app/models/auth.py b/galaxy_ng/app/models/auth.py
index 0933c2b5..144c9285 100644
--- a/galaxy_ng/app/models/auth.py
+++ b/galaxy_ng/app/models/auth.py
@@ -3,6 +3,7 @@ import logging
 from django.contrib.auth import models as auth_models

 from pulpcore.plugin.models import Group as PulpGroup
+from ansible_base.resource_registry.fields import AnsibleResourceField

 log = logging.getLogger(__name__)

@@ -21,7 +22,7 @@ RH_PARTNER_ENGINEER_GROUP = f"{SYSTEM_SCOPE}:partner-engineers"
 class User(auth_models.AbstractUser):
     """Custom user model."""

-    pass
+    resource = AnsibleResourceField(primary_key_field="id")


 class GroupManager(auth_models.GroupManager):
```

```diff
diff --git a/galaxy_ng/app/api/ui/serializers/user.py b/galaxy_ng/app/api/ui/serializers/user.py
index 9d54926f..a17cb1a3 100644
--- a/galaxy_ng/app/api/ui/serializers/user.py
+++ b/galaxy_ng/app/api/ui/serializers/user.py
@@ -23,6 +23,7 @@ class GroupSerializer(serializers.ModelSerializer):

 class UserSerializer(serializers.ModelSerializer):
     auth_provider = serializers.SerializerMethodField()
+    resource = serializers.SerializerMethodField()

     class Meta:
         model = auth_models.User
@@ -37,12 +38,16 @@ class UserSerializer(serializers.ModelSerializer):
             'date_joined',
             'is_superuser',
             'auth_provider',
+            'resource',
         )
         extra_kwargs = {
             'date_joined': {'read_only': True},
             'password': {'write_only': True, 'allow_blank': True, 'required': False}
         }

+    def get_resource(self, user):
+        return user.resource.summary_fields()
+
     def get_auth_provider(self, user):
         if hasattr(user, 'social_auth') \
                 and 'social_django' in settings.INSTALLED_APPS \
```

The same thing should work for the Teams and Orgs APIs (though I don't know that we ever formally added APIs for those things, so they may have to be net new).